src/rendering/RenderUtil.cc

```diff
@@ -2058,6 +2058,27 @@ void RenderUtilPrivate::LowlightNode(const rendering::NodePtr &_node)
   }
 }
 
+/////////////////////////////////////////////////
+void RenderUtil::ScaleWireBox(const Entity &_entityId,
+    const math::Vector3d &_scale)
+{
+  if (this->dataPtr->wireBoxes.find(_entityId) ==
+          this->dataPtr->wireBoxes.end())
```

Comment:
I think the issue is that we're not tracking wireboxes in the `sceneManager`. I added a note to do it in the future if you think it's OK.

Reply:
If I understand correctly, this function is getting `entityId`  -> `wirebox` -> `parent visual`, and setting the scale on the parent visual.

The `NodeById` function should give you the parent visual directly from the entity ID. So I'm not sure why we need to go through the wirebox.

The wirebox is created at runtime when the visual is highlighted, so it may be a less reliable way of getting to the visual.